sys/netgraph/ng_pppoe.c

```diff
@@ -1504,8 +1506,8 @@ ng_pppoe_rcvdata(hook_p hook, item_p item)
 		 * Start working out the tags to respond with.
 		 */
 		uniqtag.hdr.tag_type = PTT_AC_COOKIE;
-		uniqtag.hdr.tag_len = htons((u_int16_t)sizeof(sp));
-		uniqtag.data.pointer = sp;
+		uniqtag.hdr.tag_len = htons((u_int16_t)sizeof(uniqtag.data.pointer));
+		uniqtag.data.pointer = (ptraddr_t)(uintptr_t)sp;
```

Comment:
I guess renaming this to be more correct is more effort than you'd like :)

Reply:
Well, and a larger diff relative to upstream.

Reply:
Yeah I guess you can't really upstream the name change without a `ptraddr_t` upstream to change the type :(